### ISSUE TITLE
Satellite API: Page Number button does not return page number

### ISSUE THREAD

**Issue by downhall** [bitfocus/companion-satellite#131]
In the Satellite API the PAGENUM KEY-STATE does not return a text value. The page number value would be helpful here to understand which page was currently selected.

"KEY-STATE DEVICEID=10401 KEY=8 TYPE=PAGENUM  COLOR=#000000 TEXT= FONT_SIZE=undefined PRESSED=false"

Companion: 3.4.0, OS: Ubuntu: 22:04 server

**Comment by dnmeid** [comment]
Yes, it should give you the text of the page number button (either number or name).
As a workaround you can use a regular button instead and just type the page number on it.
